sql2.txt

```diff
@@ -17,29 +29,56 @@ may have ordered the same product in several different orders.  So the total qua
 SUM(Quantity) AS TotalQuantity and you must GROUP BY CustomerID, ProductID.  Paste your
 SQL statement below.
 
-
+SELECT c.CustomerName, p.ProductName, SUM(od.Quantity) AS TotalQuantity
+FROM Customers c
+JOIN Orders o ON c.CustomerID = o.CustomerID
+JOIN OrderDetails od ON o.OrderID = od.OrderID
+JOIN Products p ON od.ProductID = p.ProductID
+GROUP BY c.CustomerID, p.ProductID, p.ProductName
```

Comment:
The question says you must GROUP BY CustomerID, ProductID.. you are not required to add p.ProductName. kindly fix it